### ISSUE TITLE
How to deal with components without styles but with attrs

### ISSUE THREAD

**Issue by ezhlobo** [styled-components/stylelint-processor-styled-components#192]
When we use `block-no-empty` rule and write code like:

```js
export const FilterDivider = styled(Divider).attrs({
  color: COLOR_DIVIDER,
})``
```

It raises a warning. It's expected behavior and I see two options to resolve my issue:
1. Turn off `block-no-empty` rule
2. Add inline comments to each case

Do you think we can not raise warning in cases when we pass just attributes without defining styles?

**Comment by Voronar** [comment]
The same thing for this case:
```js
export const StyledSection = styled(FlexColumn)``;
```
